tests/pipelines/test_config_runner.py

```diff
@@ -14,9 +16,14 @@
 
 class CustomConfigRunner(dl.SupervisedConfigRunner):
     def get_datasets(self, stage: str) -> "OrderedDict[str, Dataset]":
+        params = deepcopy(self._stage_config[stage]["loaders"]["datasets"])
```

Comment:
is it okay, that we have to use `params = deepcopy(self._stage_config[stage]["loaders"]["datasets"])`?
from the users perspective?

Reply:
Either `deepcopy` , or we should add method `get_stage_config`. Otherwise, there always be a chance of overwriting config params.